packages/eslint-plugin-turbo/tsconfig.json

```diff
@@ -0,0 +1,17 @@
+{
+  "exclude": ["templates"],
+  "compilerOptions": {
+    "lib": ["ES2019"],
```

Comment:
curious, is this a standard we have across the board? Where does it come from and how does it get updated?

Reply:
We do not have a standard for this with Turborepo.

Reply:
it's not an established standard, but this is mostly lifted from our other packages in an effort to stay consistent. But we should practice what we preach and abstract this.